tests/Double/Stub/Transaction/ClassWithTransactionAttribute.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenClassrooms\ServiceProxy\Tests\Double\Stub\Transaction;
+
+use OpenClassrooms\ServiceProxy\Attribute\Transaction;
+
+class ClassWithTransactionAttribute
+{
```

Comment:
we should delete the old test class with annotations

Reply:
done, thx.